packages/plugin-credentials/src/credentials/common.js

```diff
@@ -357,16 +377,16 @@ export default {
    */
   set(attrs, options) {
     if (isObject(attrs)) {
+      if (attrs.authorization) {
+        attrs = attrs.authorization;
+      }
+
       if (attrs.access_token) {
```

Comment:
authorization, if a result of the spark-core credentials plugin will have an access_token. if it's a result of the legacy sdk credentials layer, it'll have a supertoken. Basically, these if statements cover all the shapes i can think of.

(I've told a few people this lately: getting the credentials api right is one of the major blockers for releasing 1.0.0)